modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala

```diff
@@ -42,8 +44,13 @@ object Databricks {
 
           def extendTable(info: ShreddedType.Info): Option[Block] = None
 
-          def getLoadStatements(discovery: DataDiscovery): LoadStatements =
-            NonEmptyList.one(Statement.EventsCopy(discovery.base, discovery.compression, getColumns(discovery)))
+          def getLoadStatements(discovery: DataDiscovery, eventsColumns: List[String]): LoadStatements = {
+            val toCopy = getColumns(discovery)
+            val toSkip = eventsColumns
+              .filter(c => c.startsWith(UnstructPrefix) || c.startsWith(ContextsPrefix))
+              .diff(toCopy)
+            NonEmptyList.one(Statement.EventsCopy(discovery.base, discovery.compression, toCopy, toSkip))
+          }
 
           def getColumns(discovery: DataDiscovery): List[String] = {
```

Comment:
It seems it's already been here before, but is it the same code we have in Snowflake? Maybe it could go to some common for discovery?